Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org'
 
 gemspec
+# this cannot go into gemspec, bundler_ext fails to load this gem.
+# see http://projects.theforeman.org/issues/16760
+gem 'concurrent-ruby', '~> 1.0', require: 'concurrent'
```

Comment:
This is related to difference between bundler and bundler_ext in handling loading errors. The ticket has the details.